src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -242,4 +244,5 @@ namespace AppInstaller::Settings
     const UserSettings* TryGetUser();
     UserSettings const& User();
     bool TryInitializeCustomUserSettings(std::string content);
+    Manifest::ScopeEnum ConvertToScopeEnum(Settings::ScopePreference scope);

```

Comment:
I think previously we had separate scope enum for settings, so we don't mix manifest concept with settings, but if we are dealing with Manifest::ScopeEnum here, should we just return Manifest::ScopeEnum directly so there's no need of SettingsScopeEnum?